rllib/core/rl_module/torch/torch_rl_module.py

```diff
@@ -1,25 +1,135 @@
 import pathlib
+import sys
+from dataclasses import dataclass, field
 from typing import Any, List, Mapping, Tuple, Union, Type
-
 from ray.rllib.core.rl_module.rl_module_with_target_networks_interface import (
     RLModuleWithTargetNetworksInterface,
 )
 from ray.rllib.core.rl_module import RLModule
-from ray.rllib.models.distributions import Distribution
+from ray.rllib.models.torch.torch_distributions import TorchDistribution
 from ray.rllib.utils.annotations import override
 from ray.rllib.utils.framework import try_import_torch
 from ray.rllib.utils.typing import NetworkType
 
 torch, nn = try_import_torch()
 
 
+@dataclass
```

Comment:
If we have torch import issues because of this being here, we should move this class to it's own module that doesn't import other stuff.